src/src/openthread-core-efr32-config.h

```diff
@@ -41,19 +41,31 @@
 #ifndef OPENTHREAD_CORE_EFR32_CONFIG_H_
 #define OPENTHREAD_CORE_EFR32_CONFIG_H_
 
+// Use (user defined) application config file to define OpenThread configurations
+#ifdef SL_OPENTHREAD_APPLICATION_CONFIG_FILE
+#include SL_OPENTHREAD_APPLICATION_CONFIG_FILE
+#endif
+
+// Use (pre-defined) stack features config file available for applications built
+// with Simplicity Studio
+#ifdef SL_OPENTHREAD_STACK_FEATURES_CONFIG_FILE
+#include SL_OPENTHREAD_STACK_FEATURES_CONFIG_FILE
+#endif
+
 /**
  * @def OPENTHREAD_CONFIG_LOG_OUTPUT
  *
  * The efr32 platform provides an otPlatLog() function.
  */
-#ifndef OPENTHREAD_CONFIG_LOG_OUTPUT /* allow command line override */
+#ifndef OPENTHREAD_CONFIG_LOG_OUTPUT
 #define OPENTHREAD_CONFIG_LOG_OUTPUT OPENTHREAD_CONFIG_LOG_OUTPUT_PLATFORM_DEFINED
 #endif
 
 /*
  * @def OPENTHREAD_CONFIG_RADIO_915MHZ_OQPSK_SUPPORT
  *
  * Define to 1 if you want to enable physical layer to support OQPSK modulation in 915MHz band.
+ * (currently not supported).
```

Comment:
Hi @lmnotran  - Can we delete this change which says "currently not supported" ? This option is supported in GitHub.